### PR TITLE
fix: the flake8 max line failure E501 and some other errors

### LINKED ISSUE #2111: python-flake8 max-line-length failure
The setup.cfg defines a max-line-length = 120 and this is overwritten in the python-flake8 with 127.

The extra definition in the worker script should be removed so that the settings from setup.cfg are used

### PR DESCRIPTION
**Pull request by rohit2p**
**Purpose of PR?**:

Fixes #2111 
**If the changes in this PR are manually verified, list down the scenarios covered:**:

Before
Untitled

After
Untitled

**Checklist:**
- [X] Bug fix. Fixes #2111 

**This pull request addresses several flake8 errors and undefined name issues in the codebase. I have adjusted the indentation, comments, and variable definitions to comply with the flake8 style guide. Additionally, i removed the max-line-length = 127 from the python-flake8 because it was overwriting setup.cfg. These changes ensure that the codebase adheres to the project's coding standards and resolves the reported issues.**

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please send the pull request to develop.

and we don't test docs for flake8 because this is automated created, we can't change that.

File: .github/workflows/python-flake8.yml [hunk 1]
```diff
@@ -31,4 +31,4 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8
-        flake8 --count --max-line-length=127 --statistics mslib tests
+        flake8 --count --statistics mslib tests
```
Comment (ReimarBauer) [inline .github/workflows/python-flake8.yml:34]: we don't run it on docs, see parameters.

**Comment by rohit2p** [comment]
at this page https://open-mss.github.io/develop/GSOC/ideas i read that all bug fix go to stable branch that's why i did that😔

File: .github/workflows/python-flake8.yml (hunk 1 above)
Comment (ReimarBauer) [reply .github/workflows/python-flake8.yml:34]: when you fixed docs now, then extend the parameters.

But I am not sure if that makes sense, when make docs is run, then other py scripts will become created below docs, which are automated done.

Comment (rohit2p) [reply .github/workflows/python-flake8.yml:34]: What do i do now then?

**Comment by ReimarBauer** [comment]
> at this page https://open-mss.github.io/develop/GSOC/ideas i read that all bug fix go to stable branch that's why i did that😔

obviously you have not created a branch from stable but from develop. 

When you want to send it to stable, use the right branch from the beginning ;) 

A fix should not get all changes of develop into stable ;)

**Comment by ReimarBauer** [comment]
Do you want to redo for stable?

File: .github/workflows/python-flake8.yml (hunk 1 above)
Comment (ReimarBauer) [reply .github/workflows/python-flake8.yml:34]: you did a bit too much, as in the "good first" issue was defined, read line 34, extend it to the docs dir too.

But when you do this the error will block its merging. so it must be solved too. And that gets then easily into not been a good first issue.

So leave it as it is.

**Comment by rohit2p** [comment]
yes i can do that 
but i have a question ?
like when i do that should i leave this part as it is 
```
      run: |
        python -m pip install --upgrade pip
        pip install flake8
        flake8 --count --max-line-length=127 --statistics mslib tests
```
and only make changes in the work script ??
i dont't get it

File: .github/workflows/python-flake8.yml (hunk 1 above)
Comment (rohit2p) [reply .github/workflows/python-flake8.yml:34]: ok understood. actually its my first tym doing a contribution that way i got confused.
so here is the thing i think i should do now say yes and if i understood the assignment correctly.
i will fix all the issue in #2111 but this time from the stable branch.
and i will leave the python-flake8 file as it is
?

Comment (ReimarBauer) [reply .github/workflows/python-flake8.yml:34]: don't touch the conf.py - no changes. Only remove the extra parameter from the CI file.

Look also in my comment in the slack channel

Comment (ReimarBauer) [reply .github/workflows/python-flake8.yml:34]: the reason for _good first issue_, is to get a simple thing done, by learning the workflow. There bigger other issues later.

**Comment by ReimarBauer** [comment]
> i dont't get it

only this

only_this

**Comment by ReimarBauer** [comment]
Now we have 2 PR for this?

**Comment by rohit2p** [comment]
yea but i closed it since in this the branch was created from develop and i did a bit too much so i redid from stable

**Comment by ReimarBauer** [comment]
Ok

**Pull request closed without merging**